hack/simple-demo.sh

```diff
@@ -19,11 +19,28 @@
 # See https://github.com/enterprise-contract/golden-container
 IMAGE=${IMAGE:-"quay.io/redhat-appstudio/ec-golden-image:latest"}
 
+# Assume the latest image was pushed already
+GIT_REPO=enterprise-contract/golden-container
+GIT_SHA=$(curl -s "https://api.github.com/repos/${GIT_REPO}/commits?per_page=1" | jq -r '.[0].sha')
```

Comment:
Could read the `vcs-ref` label of the image in case the latest commit hasn't been pushed:

```suggestion
GIT_SHA=$(skopeo inspect docker://quay.io/redhat-appstudio/ec-golden-image:latest --format '{{index .Labels "vcs-ref"}}')
```